src/components/reusable/notification/notificationPanel.ts

```diff
@@ -0,0 +1,73 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import '@kyndryl-design-system/shidoka-foundation/components/button';
+
+import NotificationPanelScss from './notificationPanel.scss';
+
+/**
+ * Notification Panel.
+ * @slot menu-slot - Slot for panel menu
+ * @slot unnamed - Slot for notification content.
+ * @fires on-footer-btn-click - Emits the panel footer button event.
+ */
+
+@customElement('kyn-notification-panel')
```

Comment:
Sine it's now a sub-component of Header, move this component to the `header` folder, and change the tag name to `kyn-header-notification-panel`.

Reply:
yes seems it sound well.

Reply:
Done